Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-columns-selector.js

```diff
@@ -0,0 +1,36 @@
+angular.module('virtoCommerce.exportModule')
+    .controller('virtoCommerce.exportModule.exportColumnsSelectorController', ['$scope', 'platformWebApp.bladeNavigationService', 'virtoCommerce.exportModule.exportModuleApi', function ($scope, bladeNavigationService, exportApi) {
```

Comment:
Seems `'virtoCommerce.exportModule.exportModuleApi'` = `exportApi` is not used here